catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/taxonomy/Core.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.data.taxonomy;
+
+/**
+ * <p>
+ * <b> This code is experimental. While this interface is functional and tested, it may change or be
+ * removed in a future version of the library. </b>
+ * </p>
+ */
+public interface Core {
+    /**
+     * {@link ddf.catalog.data.Attribute} checksum value for the {@link ddf.catalog.data.Metacard#RESOURCE_URI}
+     */
+    String CHECKSUM = "checksum";
+
+    /**
+     * {@link ddf.catalog.data.Attribute} algorithm used to calculate the checksum on the {@link ddf.catalog.data.Metacard#RESOURCE_URI}
+     * for local resources
+     */
+    String CHECKSUM_ALGORITHM = "checksum-algorithm";
+
+    /**
+     * {@link ddf.catalog.data.Attribute} name for accessing the date/time this {@link ddf.catalog.data.Metacard} was created. <br/>
```

Comment:
This is for when the product was created.  There is another attribute for when the Metacard was created.